Uno.Gallery/Uno.Gallery.Mobile/Android/Main.Android.cs

```diff
@@ -22,6 +23,7 @@ namespace Uno.Gallery.Droid
 	)]
 	public class Application : Windows.UI.Xaml.NativeApplication
 	{
+		[DynamicDependency(DynamicallyAccessedMemberTypes.All, "Android.Text.ITextWatcherInvoker", "Mono.Android")]
```

Comment:
Is this something we can possibly set directly in the Uno.Material library or it has to be at the app level? 

Also, maybe a comment for why its needed and a link to https://github.com/xamarin/xamarin-android/issues/7097